ocs_ci/deployment/helpers/storage_class.py

```diff
@@ -0,0 +1,64 @@
+import logging
+import yaml
+
+from ocs_ci.framework import config
+from ocs_ci.framework.logger_helper import log_step
+from ocs_ci.ocs import constants
+from ocs_ci.utility.utils import run_cmd
+
+
+logger = logging.getLogger(__name__)
+
+DEFAULT_STORAGE_CLASS_MAP = {
+    constants.AWS_PLATFORM: "gp2-csi",
+    constants.IBMCLOUD_PLATFORM: "ibmc-vpc-block-10iops-tier",
+    constants.VSPHERE_PLATFORM: "thin-csi",
+    constants.AZURE_PLATFORM: "managed-csi",
+    constants.GCP_PLATFORM: None,
+    constants.ROSA_HCP_PLATFORM: None,
+    constants.RHV_PLATFORM: "ovirt-csi-sc",
+}
```

Comment:
The `thin-csi` value is also available as `constants.THIN_CSI_STORAGECLASS`, but I'm not sure, if it is worth to change all the values and get them from constants, so maybe as it is now is ok.